firefox-ios/Client/Coordinators/CoordinatorFlagManager.swift

```diff
@@ -10,4 +10,8 @@ struct CoordinatorFlagManager {
     static var isQRCodeCoordinatorEnabled: Bool {
         return NimbusManager.shared.featureFlagLayer.checkNimbusConfigFor(.qrCodeCoordinatorRefactor)
     }
+    
+    static var isBackForwardListShownFromCoordaintorEnabled: Bool {
```

Comment:
Should we be adding this if we're not using it? Furthermore, @lmarceau, if we're adding a layer of this type, there's no reason to add this to the `FeatureFlagLayer`. Was there a reason both exist? I don't remember. I'd prefer there be a single source of truth into nimbus for any feature.

Reply:
@adudenamedruby I added it, because I ll use it for the next pr. But I can't put it in the next or too

Reply:
@adudenamedruby We've been using those static var for flags on coordinator as normally coordinators need to be check in multiple places. It avoids having to check for `NimbusManager.shared.featureFlagLayer.checkNimbusConfigFor(.backForwardListCoordinatorRefactor)` in multiple places, but if that's to be avoided then any other way if fine by me. 

As for `there's no reason to add this to the FeatureFlagLayer` -> I am not sure to understand, as any flags I added in the past were always added to that layer 😬 So if there's another way, I'm unsure how.